quickstart.md

```diff
@@ -759,14 +759,19 @@ Run skopeo to encrypt the image created in the previous section:
 
 ```
 sudo OCICRYPT_KEYPROVIDER_CONFIG=$(pwd)/attestation-agent/sample_keyprovider/src/enc_mods/offline_fs_kbs/ocicrypt.conf \
-skopeo copy --insecure-policy docker-daemon:[REGISTRY_URL]:unencrypted \
-docker-daemon:[REGISTRY_URL]:encrypted \
+skopeo copy --insecure-policy \
+docker-daemon:[REGISTRY_URL]:unencrypted \
+docker:[REGISTRY_URL]:encrypted \
 --encryption-key provider:attestation-agent:$(pwd)/keys.json:key_id1
 ```
 
 Again, be sure to replace `[REGISTRY_URL]` with the desired registry URL.
 `--insecure-policy` flag is used to connect to the attestation agent and will not impact the security of the project.
 
+Make sure to use the `docker` target in the destination URL in the `skopeo copy` command as demonstrated above. Publishing the 
+encrypted image to the local `docker-daemon` is known to have issues and not provide an adequate error response. It is better to 
+publish directly to a remote registry known to support encrypted images like GitHub Container Registry (GHCR).
+
 At this point it is a good idea to inspect the image was really encrypted as skopeo can silently leave it unencrypted. Use
```

Comment:
@ryansavino you will need to rephrase that.

Reply:
ah, and also below when you read "Push the encrypted image to the registry:"...it doesn't need to push the image to docker as skopeo does it already.

Reply:
Check it out now, and let me know if it reads okay.